package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "frau-jwt",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

Reverting since I think this is a patch instead of minor, so will do it outside of the PR.

```suggestion
  "version": "2.2.0",
```